homeassistant/components/brother/config_flow.py

```diff
@@ -35,12 +40,14 @@ class BrotherConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
 
-    def __init__(self):
+    def __init__(self) -> None:
         """Initialize."""
-        self.brother = None
-        self.host = None
+        self.brother: Brother = None
+        self.host: str | None = None
 
-    async def async_step_user(self, user_input=None):
+    async def async_step_user(
+        self, user_input: ConfigType | None = None
```

Comment:
`ConfigType` is intended for use in `async_setup` method. Although type is the same, underlying entity is different. Please use just `dict[str, Any]` here.